samples/unreal/07_amass.py

```diff
@@ -0,0 +1,105 @@
+"""
+>>> python -m samples.unreal.07_amass
+
+This is a script to demonstrate importing Amass motion and applying it to SMPL-XL model.
+Before running this script, please download `SMPL-XL model` and `Amass dataset` first,
+you can find the download links in the comments in main function.
+
+SMPL-XL: a parametric human model based on SMPL-X in a layered representation, introduced in https://synbody.github.io/
+Amass: a large database of human motion, introduced in https://amass.is.tue.mpg.de/
+"""
+from pathlib import Path
+
+import xrfeitoria as xf
+from xrfeitoria.data_structure.models import RenderPass
+from xrfeitoria.utils import setup_logger
+from xrfeitoria.utils.anim import dump_humandata, load_amass_motion
+
+from ..config import unreal_exec, unreal_project
+
+# prepare the assets
+####################
+root = Path('.cache/sample-amass').resolve()  # modify this to your own path
+
+# 1. Download Amass from https://amass.is.tue.mpg.de/download.php
+# For example, download ACCAD (SMPL-X N) from https://download.is.tue.mpg.de/download.php?domain=amass&sfile=amass_per_dataset/smplx/neutral/mosh_results/ACCAD.tar.bz2
+# and use `ACCAD/s001/EricCamper04_stageii.npz` from the uncompressed folder
+amass_file = root / 'EricCamper04_stageii.npz'
+
+# 2.1 Download SMPL-XL model from https://openxrlab-share.oss-cn-hongkong.aliyuncs.com/xrfeitoria/assets/SMPL-XL-001.fbx
+# or from https://openxlab.org.cn/datasets/OpenXDLab/SynBody/tree/main/Assets
+# With downloading this, you are agreeing to CC BY-NC-SA 4.0 License (https://creativecommons.org/licenses/by-nc-sa/4.0/).
+smpl_xl_file = root / 'SMPL-XL-001.fbx'
+# 2.2 Download the meta information from https://openxrlab-share.oss-cn-hongkong.aliyuncs.com/xrfeitoria/assets/SMPL-XL-001.npz
+smpl_xl_meta_file = root / 'SMPL-XL-001.npz'
+
+# 3. Define the output file path
+seq_name = 'seq_amass'
+output_path = Path(__file__).parents[2].resolve() / 'output/samples/unreal' / Path(__file__).stem
```

Comment:
```suggestion
output_path = Path(__file__).resolve().parents[2] / 'output/samples/unreal' / Path(__file__).stem
```

Reply:
Accepted in [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816#diff-050f51ccfd09bce14b1e89a956edfd76f9a5097030783dc6c6970db03c6e208fR38)

Reply:
Modified all the similar situations [`17a3e7a` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/17a3e7aec1fb07bb619b36e2739adad91903b6d9)